scrapy_poet/commands.py

```diff
@@ -83,7 +83,12 @@ def run(self, args, opts):
 
         spider_cls = spider_for(cls)
         self.settings["ITEM_PIPELINES"][SavingPipeline] = 100
+        self.settings["DOWNLOADER_MIDDLEWARES"][
+            "scrapy_poet.downloadermiddlewares.InjectionMiddleware"
+        ] = None
+        self.settings["DOWNLOADER_MIDDLEWARES"][InjectionMiddleware] = None
         self.settings["DOWNLOADER_MIDDLEWARES"][SavingInjectionMiddleware] = 543
+        self.settings["_SCRAPY_POET_SAVEFIXTURE"] = True
```

Comment:
I wonder if it'd be simpler if we have a single InjectionMiddleware, which reads the value of _SCRAPY_POET_SAVEFIXTURE setting and acts accordingly.